src/components/LeftDrawer/LeftDrawer.tsx

```diff
@@ -5,7 +5,6 @@ import { useHistory } from 'react-router-dom';
 import { ReactComponent as AngleRightIcon } from 'assets/svgs/angleRight.svg';
 import { ReactComponent as LogoutIcon } from 'assets/svgs/logout.svg';
 import { ReactComponent as OrganizationsIcon } from 'assets/svgs/organizations.svg';
-import { ReactComponent as RequestsIcon } from 'assets/svgs/requests.svg';
```

Comment:
Check whether this icon is used somewhere else or not, if not then remove this svg file too

Reply:
Ok, I'll do that.